### PR TITLE
chart panel & ui middleware

### PR DESCRIPTION
**Pull request by AntonNazarenko**

### CODE REVIEW

File: src/redux/middleware/ui/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import WSActions from '../../actions/ws'
+import UITypes from '../../constants/ui'
+
+export default () => {
+  return store => next => (action = {}) => {
+    const { type } = action
+
+    switch (type) {
+      case UITypes.SAVE_LAYOUT: {
+        store.dispatch(WSActions.recvNotification({
```
Comment (robertkowalski) [inline src/redux/middleware/ui/index.js:10]: it reads a bit strange to use a websocket action for local notifciations

Comment (robertkowalski) [reply src/redux/middleware/ui/index.js:10]: do we have a local alternative?

Comment (AntonNazarenko) [reply src/redux/middleware/ui/index.js:10]: no, I can create it if needed, it would be doing the same thing, but it would be more clear for developers

**Comment by robertkowalski** [comment]
after building the css i get a lot of white bars

Screen Shot 2020-10-12 at 18 36 14

File: src/redux/middleware/ui/index.js (hunk 1 above)
Comment (robertkowalski) [reply src/redux/middleware/ui/index.js:10]: if we don't refactoring it to be a `ui` action would make sense i guess... `recvNotification` has not many usages, so should be a quick change...

**Comment by robertkowalski** [comment]
the fields over the chart changed.

current:
Screen Shot 2020-10-12 at 18 40 23


new:
Screen Shot 2020-10-12 at 18 37 29

**Review by robertkowalski — CHANGES_REQUESTED** [review]
this needs some fine tuning and fixes

**Comment by robertkowalski** [comment]
field issue fixed by rebase

File: src/redux/reducers/notifications/index.js [hunk 2]
```diff
@@ -9,9 +10,10 @@ function reducer(state = getInitialState(), action = {}) {
   const { type, payload = [] } = action
 
   switch (type) {
-    case types.DATA_NOTIFICATION: {
+    case UITypes.DATA_NOTIFICATION:
+    case WSTypes.DATA_NOTIFICATION: {
       const { notification = [] } = payload
-
+      console.log(notification)
```
Comment (robertkowalski) [inline src/redux/reducers/notifications/index.js]: debug?

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**